README.md

```diff
@@ -1,48 +1,26 @@
-# Hydro technical spec
+# What is Hydro?
 
-This is an overview of the core smart contracts that make up Hydro as of now.
-Not all features from the original [litepaper](https://forum.cosmos.network/t/atom-wars-introducing-the-hydro-auction-platform/13842) are present in this version,
-and some implementation details might lead to slightly different behavior than described in the litepaper.
+Hydro is a auction platform where ATOM stakers can lock up their staked Atoms in exchange for voting power, and use it to vote on proposals to deploy tokens owned by the community pool into liquidity (aka market-making) positions on various decentralized exchanges. The liquidity itself is owned by the Cosmos Hub (although initially custodied by a committee), thus this is a way of deploying Protocol Owned Liquditiy (PoL). The advantage of PoL compared to issuing incentives (i.e. paying rewards to liquidity providers in certain pools) is that the community pool does not *spend* its funds, but rather deploys them. This means that the funds can be re-used in the future, and the community pool can grow over time. Projects using the Cosmos Hub’s Interchain Security platform receive several beenfits (special tranches, longer export duration etc.)
 
-# Protocol Owned Liquidity
 
-Hydro is a system where Atom stakers can lock up their staked Atoms in exchange for voting power, which they can use to vote on proposals to deploy Atoms owned by the community pool into liquidity (aka market-making) positions on various decentralized exchanges.
-The liquidity itself is still owned by the Cosmos Hub, thus this is a way of deploying Protocol Owned Liquditiy (PoL).
+Hydro is implemented based on its [litepaper](https://forum.cosmos.network/t/atom-wars-introducing-the-hydro-auction-platform/13842). That said, we've already incorporated and will continue to incorporate feedback, which means that implementation will increasingly diverge from the paper. Up-to-date Hydro documentation will be posted on the Hydro website.
```

Comment:
Do we have a link to the Hydro website? Even if it's just a message "coming soon".